_tsml_research_resources/soton/iridis/iridis_python.md

```diff
@@ -143,16 +145,22 @@ For CPU experiments start with one of the following scripts:
 
 >clustering_experiments.sh
 
+Within the above scripts you will need to adjust the top half section to work for your account. All the steps are described as comments within the script.
+
 You may need to use `dos2unix` to convert the line endings to unix format.
 
 The default queue for CPU jobs is _batch_.
 
+You can simply run `sh classification_experiments.sh` or alternativley write a slurm script to request the job. An example slurm script is found in this directory for running classification_experiments.sh.
```

Comment:
I don't recommend this

Reply:
Does the classification_experiments.sh have all the equivalent commands as a SLURM script? The iridis wiki only mentions slurm scripts in the 'batch jobs section' https://sotonac.sharepoint.com/teams/HPCCommunityWiki/SitePages/Connecting-to-Iridis5.aspx. If these are equivalent then ofc the slurm would be redundant.

Reply:
You dont need to create a separate job to run the script, you can just run in an interactive or on the login node.